### PR TITLE
Add tbtamr 2

### PR DESCRIPTION
**Pull request by kristyhoran**
Describe your pull request here

Addition of tbtamr a new package. Addressed licence issue raised previously

### CODE REVIEW

File: recipes/tbtamr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "{{ sha256 }}"
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - tbtamr=tbtamr.tbtamr:main
+  run_exports:
+    - {{ pin_subpackage('tbtamr', max_pin="x.x.x") }}
```
Comment (thanhleviet) [inline recipes/tbtamr/meta.yaml:19]: Will the patch release break things?
| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |

Comment (kristyhoran) [reply recipes/tbtamr/meta.yaml:19]: @thanhleviet thank you .. do i need to make this change or have you made it and then will accept?

Comment (thanhleviet) [reply recipes/tbtamr/meta.yaml:19]: Either you or I can do it, what max_pin do you want to change then? I'd go for `x.x` :)

Comment (kristyhoran) [reply recipes/tbtamr/meta.yaml]: @thanhleviet done!! Thanks for your help.

**Review by thanhleviet — APPROVED** [review]

**Pull request merged**